packages/peregrine/lib/talons/MiniCart/useProduct.js

```diff
@@ -0,0 +1,45 @@
+import { useCallback, useState } from 'react';
+
+const imageWidth = 80;
+const imageHeight = 100;
+
+export const useProduct = props => {
+    const { beginEditItem, item, removeItemFromCart } = props;
+    const { image, name, options, price, qty } = item;
+    const [isFavorite, setIsFavorite] = useState(false);
+    const [isLoading, setIsLoading] = useState(false);
+
+    const handleFavoriteItem = useCallback(() => {
+        setIsFavorite(!isFavorite);
+    }, [isFavorite]);
+
+    const handleEditItem = useCallback(() => {
+        beginEditItem(item);
+    }, [beginEditItem, item]);
+
+    const handleRemoveItem = useCallback(() => {
+        setIsLoading(true);
+        removeItemFromCart({ item });
+    }, [item, removeItemFromCart]);
+
+    const imageObj = {
+        url: image.file,
```

Comment:
Interesting—any reason to change from `file` to `url`? Not that I mind necessarily.

Reply:
No, it's just the syntax we used in `useCategoryTile` so I'm trying to be consistent.